app/helpers/application_helper.rb

```diff
@@ -154,13 +155,21 @@ def ox_card(classes: "", heading: "", &block)
         end
       end
 
+      banners_divs = if banners.present?
```

Comment:
I would suggest using `unless banners.empty?` just because `present?` is a Rails extension, not standard Ruby.

But actually you should be able to just remove this check because an empty array should make banners_divs a perfectly fine blank string.

Reply:
True. Done. I removed that check as you said.